docs/end-user-dev-kubeconifg.md

```diff
@@ -0,0 +1,32 @@
+# User admin for development
+
+There can be situations where you as a developer needs to act as a end user.
```

Comment:
We decided to use End User to denote "Ultimate user of the Application deployed on top of Kubernetes.". Can you harmonize with our Glossary?

Reply:
Right, it should be application developer and not end user. Will fix that :+1: